libred/CMakeLists.txt

```diff
@@ -16,6 +16,32 @@ check_symbol_exists(_NSGetEnviron crt_externs.h HAVE_NSGETENVIRON)
 
 find_package(Threads REQUIRED)
 
```

Comment:
This information is duplicated here, in config.h.in, and on line 526 of libred/red.c. Is it worth writing it in a YAML file and generating the source from that as a pre-pre-build step (either using jinja or even sed or something)?